packages/ui/components/ui/tx/view/transaction.tsx

```diff
@@ -19,10 +29,7 @@ export const TransactionViewComponent = ({ txv }: { txv: TransactionView }) => {
         ))}
       </ViewSection>
       <ViewSection heading='Parameters'>
-        <ViewBox
-          label='Transaction Fee'
-          visibleContent={<div className='font-mono'>{fee} upenumbra</div>}
-        />
+        <ViewBox label='Fee' visibleContent={<div className='font-mono'>{fee} UM</div>} />
```

Comment:
Consider whether we can use `ValueViewComponent` here given that we have an `Metadata` and `Amount`

Reply:
thinking about something like this might work

```
  let feeValueView = new ValueView({
    valueView: {
      case: 'knownAssetId',
      value: { amount: txv.bodyView.transactionParameters.fee.amount, metadata: feeAssetMetadata },
    },
  });
```

and `<ViewBox label='Fee' visibleContent={<ValueViewComponent view={feeValueView} />} />`

Reply:
<img width="374" alt="Screenshot 2024-06-16 at 9 29 45 PM" src="https://github.com/penumbra-zone/web/assets/70081547/0438d825-b336-4ae9-a4f8-6a6e9fc7eded">